services/notify/notifier.go

```diff
@@ -77,4 +78,6 @@ type Notifier interface {
 	ChangeDefaultBranch(ctx context.Context, repo *repo_model.Repository)
 
 	CreateCommitStatus(ctx context.Context, repo *repo_model.Repository, commit *repository.PushCommit, sender *user_model.User, status *git_model.CommitStatus)
+
+	CreateWorkflowJob(ctx context.Context, repo *repo_model.Repository, sender *user_model.User, job *actions_model.ActionRunJob, task *actions_model.ActionTask)
```

Comment:
> This event occurs when there is activity relating to a job in a GitHub Actions workflow.

This notifier should be triggered when the status changed, so that I think the function name is not right to express itself.